action.yaml

```diff
@@ -9,12 +9,18 @@ inputs:
     required: false
     default: ''
     description: Space separated packages to install via dnf install (can contain options)
+  workdir:
+    required: false
+    default: ''
```

Comment:
I needed to check how this works. By default, this is set to an empty string and passed as an empty string to WORKDIR. On the other end: https://github.com/fedora-python/fedora-python-tox/blob/e5a0a5b81cdd42019f01c9671827dad0b9d1ae81/run_tests.sh#L19 WORKDIR is checked with `-z` which means check for emtynes. Alright then.